windows_containers/wmco_rn/windows-containers-release-notes-10-15-x.adoc

```diff
@@ -14,18 +14,17 @@ The release notes for Red{nbsp}Hat OpenShift for Windows Containers tracks the d
 
 Starting with this release, y-stream releases of the WMCO will be in step with {product-title}, with only z-stream releases between {product-title} releases. The WMCO numbering reflects the associated {product-title} version in the y-stream position. For example, the current release of WMCO is associated with {product-title} version 4.15. Thus, the numbering is WMCO 10.15.z.
 
-[id="wmco-10-15-2"]
-== Release notes for Red{nbsp}Hat Windows Machine Config Operator 10.15.2
+[id="wmco-10-15-3"]
+== Release notes for Red{nbsp}Hat Windows Machine Config Operator 10.15.3
 
-This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 10.15.2 were released in link:https://access.redhat.com/errata/RHBA-2024:2704[RHBA-2024:2704].
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 10.15.3 were released in link:https://access.redhat.com/errata/RHBA-TDB[RHBA-TBD].
```

Comment:
Based on the notes for peer reviewers, I'm assuming this link won't work until Aug. 22.